crates/nu-parser/fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
+corpus
+artifacts
+coverage
```

Comment:
With the instructions provided this needs more ignores:

```
        crates/nu-parser/fuzz/Cargo.lock
        crates/nu-parser/fuzz/out/
        crates/nu-parser/fuzz/seeds/
```

Reply:
ok